content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,178 @@
+
+Finally, this success couldn't have been possible without to the over [100 contributors](https://github.com/excalidraw/excalidraw/graphs/contributors) that helped us ship new features, improvements, and fixes right to production.
+
+https://excalidraw.com/#json=6671570797854720,bqkIFBlioHfMMLYHtcdGjA
+
+## Some of our coolest features
+
+### 🤝 Collaboration
+
+When the lockdown started, companies all around the world struggled to adapt to remote work. [idlewinn](https://github.com/idlewinn) and [petehunt](https://github.com/petehunt) [implemented live collaboration](https://github.com/excalidraw/excalidraw/pull/879) to enable it to be used for interviews, brainstorming, presentations, and more. If you're curious, we [explained how it works in a blog post](/building-excalidraw-p2p-collaboration-feature/).
+
+https://twitter.com/Vjeux/status/1238907727906127872
+
+### 🔒 Your data is encrypted
+
+Many of Excalidraw use cases involve drawing sensitive data. As such, we architected our system so that our servers never see the content of your drawings, using end-to-end encryption. The [backend](https://github.com/excalidraw/excalidraw-json) support for storing data was implemented by [lipis](https://github.com/lipis), while the [client-side encryption itself](https://github.com/excalidraw/excalidraw/pull/642) was added by [vjeux](https://github.com/vjeux). Read how we are doing it in our article on [end-to-end encryption](/end-to-end-encryption/).
+
+https://excalidraw.com/#json=5645858175451136,8w-G0ZXiOfRYAn7VWpANxw
+
+### 🇺🇳 Translations
+
+It was important for us early on to make sure that Excalidraw was translated into many languages so that it could be used all over the world. The [initial implementation](https://github.com/excalidraw/excalidraw/pull/638) was done by [fernandoalava](https://github.com/fernandoalava), while the automatic integration with our [Crowdin project](https://crowdin.com/project/excalidraw) was set up by [lipis](https://github.com/lipis). Support for right-to-left languages was [implemented](https://github.com/excalidraw/excalidraw/pull/1154) soon after by [j-f1](https://github.com/j-f1). To top it off, [Ellinor Rapp](https://www.myfonts.com/newsletters/cc/200712.html) designed new glyphs for several non-latin languages and added them to the Virgil hand-written font we're using. You can read more [about how we manage translations on the blog](/enabling-translations/)!!.
+
+### 📱 Mobile first
+
+Touch support and mobile-optimized layout [were](https://github.com/excalidraw/excalidraw/pull/787) [first](https://github.com/excalidraw/excalidraw/pull/788) [added](https://github.com/excalidraw/excalidraw/pull/790) by [j-f1](https://github.com/j-f1). This includes the creation of a toolbar for mobile devices that displays relevant controls while still leaving most of the screen free for the canvas.
```

Comment:
```suggestion
Touch support and mobile-optimized layout [was](https://github.com/excalidraw/excalidraw/pull/787) [first](https://github.com/excalidraw/excalidraw/pull/788) [added](https://github.com/excalidraw/excalidraw/pull/790) by [j-f1](https://github.com/j-f1). This includes the creation of a toolbar for mobile devices that displays relevant controls while still leaving most of the screen free for the canvas.
```

I know `were` is (also) grammatically correct, but it just doesn't roll of my tongue. Feel free to reject this if you feel it's fine.